Standards/scs-0103-v1-standard-flavors.md

```diff
@@ -0,0 +1,134 @@
+| SCS-1V-4         |      1 | shared core   |          4 |                 |            |
+| SCS-2V-8         |      2 | shared core   |          8 |                 |            |
+| SCS-4V-16        |      4 | shared core   |         16 |                 |            |
+| SCS-8V-32        |      8 | shared core   |         32 |                 |            |
+| SCS-1V-2         |      1 | shared core   |          2 |                 |            |
+| SCS-2V-4         |      2 | shared core   |          4 |                 |            |
+| SCS-4V-8         |      4 | shared core   |          8 |                 |            |
+| SCS-8V-16        |      8 | shared core   |         16 |                 |            |
+| SCS-16V-32       |     16 | shared core   |         32 |                 |            |
+| SCS-1V-8         |      1 | shared core   |          8 |                 |            |
+| SCS-2V-16        |      2 | shared core   |         16 |                 |            |
+| SCS-4V-32        |      4 | shared core   |         32 |                 |            |
+| SCS-1L-1         |      1 | crowded core  |          1 |                 |            |
+
+### Recommended
+
+| Recommended name | vCPUs  | vCPU type     | RAM [GiB]  | Root disk [GB]  | Disk type  |
+| ---------------- | ------ | ------------- | ---------- | --------------- | ---------- |
+| SCS-1V-4-10      |      1 | shared core   |          4 |              10 | (any)      |
+| SCS-2V-8-20      |      2 | shared core   |          8 |              20 | (any)      |
+| SCS-4V-16-50     |      4 | shared core   |         16 |              50 | (any)      |
+| SCS-4V-16-100s   |      4 | shared core   |         16 |             100 | ssd        |
+| SCS-8V-32-100    |      8 | shared core   |         32 |             100 | (any)      |
+| SCS-1V-2-5       |      1 | shared core   |          2 |               5 | (any)      |
+| SCS-2V-4-10      |      2 | shared core   |          4 |              10 | (any)      |
+| SCS-2V-4-20s     |      2 | shared core   |          4 |              20 | ssd        |
+| SCS-4V-8-20      |      4 | shared core   |          8 |              20 | (any)      |
+| SCS-8V-16-50     |      8 | shared core   |         16 |              50 | (any)      |
+| SCS-16V-32-100   |     16 | shared core   |         32 |             100 | (any)      |
+| SCS-1V-8-20      |      1 | shared core   |          8 |              20 | (any)      |
+| SCS-2V-16-50     |      2 | shared core   |         16 |              50 | (any)      |
+| SCS-4V-32-100    |      4 | shared core   |         32 |             100 | (any)      |
+| SCS-1L-1-5       |      1 | crowded core  |          1 |               5 | (any)      |
+
+### Guarantees and properties
+
+The following guarantees must be met:
+
+- A "shared core" means _at least 20% of a core in >99% of the time_, measured over the
+  course of one month (1% is 7,2 h/month).
```

Comment:
We had a discussion within our team, where we wondered the following: Isn't the question here, **how much is 100 %**?

There could be a big difference between `SCS-1V-4-10` on one cluster and `SCS-1V-4-10` on another, depending on the model of the CPU, even if the sharing is the same. In this vein, wouldn't it be better for the customer to know what to expect in terms of computing power?

Maybe we could introduce performance classes with respect to TFLOPs or something similar, and then x amount of TFLOPs (what the customer needs) is a function of both the CPU model and the sharing model (what the customer needn't care about).